.github/workflows/main.yml

```diff
@@ -9,6 +9,7 @@ on:
     branches:
       - main
       - 'release/*'
+  workflow_dispatch: #Necessary to allow API based trigger of the workflow
```

Comment:
Global question regarding the mechanism of this plugin and the fact that it's trigger this workflow manually:
Would it not be safer to actually create the pull request (with automerge and a dedicated label) for each cascading?
Actually the `github.context` object prorived by github is different according to the trigger of the pipeline. When we multiply the list of different triggers we risk to have issue because missing information in the context.

Reply:
Automerge but still required approvals ?

Reply:
It can be bypass actually but would it be an issue to require approvals?

Reply:
Need to test 1/ if the bypass works 2/ if the build on the destination branch is trigger